vaadin-testbench-unit-shared/src/main/java/com/vaadin/flow/component/grid/GridTester.java

```diff
@@ -313,6 +312,139 @@ private Component getRendererItem(int row, Grid.Column<Y> yColumn) {
+     * @param propertyClass
+     *            the class of the value of the LitRenderer property
+     * @param <V>
+     *            the type of the LitRenderer property
+     * @return value of renderer's property for the target cell
+     * @throws UnexpectedTypeException
+     *             when the type of the property does not match the
+     * @throws IllegalArgumentException
+     *             when column for property doesn't exist or the target column of
+     *             the cell is not a LitRenderer
+     */
+    public <V> V getLitRendererPropertyValue(int row, int column, String propertyName, Class<V> propertyClass) {
+        return getLitRendererPropertyValue(row, getColumns().get(column), propertyName, propertyClass);
+    }
+
+    private <V> V getLitRendererPropertyValue(int row, Grid.Column<Y> column,
+                                              String propertyName, Class<V> propertyClass) {
+        ensureVisible();
+        if (column.getRenderer() instanceof LitRenderer<Y> litRenderer) {
+            var valueProvider = findLitRendererProperty(litRenderer, propertyName);
+            var untypedValue = valueProvider.apply(getRow(row));
+            if (propertyClass.isInstance(untypedValue)) {
+                return propertyClass.cast(untypedValue);
+            } else {
+                throw new UnexpectedTypeException("Target column property value is the wrong type - expected %s, found %s."
+                        .formatted(propertyClass.getCanonicalName(), untypedValue.getClass().getCanonicalName()));
+            }
+        } else {
+            throw new IllegalArgumentException("Target column doesn't have a LitRenderer.");
+        }
+    }
+
+    /**
+     * Invoke named function for item's LitRenderer in column.
+     *
+     * @param row
+     *            item row
+     * @param columnName
+     *            key/property of column
+     * @param functionName
+     *            the name of the LitRenderer function to invoke
+     */
+    public void invokeLitRendererFunction(int row, String columnName, String functionName) {
+        invokeLitRendererFunction(row, getColumn(columnName), functionName);
+    }
+
+    /**
+     * Invoke named function for item's LitRenderer in column.
+     *
+     * @param row
+     *            item row
+     * @param column
+     *            column to get
+     * @param functionName
+     *            the name of the LitRenderer function to invoke
+     */
+    public void invokeLitRendererFunction(int row, int column, String functionName) {
+        invokeLitRendererFunction(row, getColumns().get(column), functionName);
+    }
+
+    private void invokeLitRendererFunction(int row, Grid.Column<Y> column, String functionName) {
+        ensureVisible();
+        if (column.getRenderer() instanceof LitRenderer<Y> litRenderer) {
+            var callable = findLitRendererFunction(litRenderer, functionName);
+            callable.accept(getRow(row), Json.createArray());
```

Comment:
JsonArray is probably something that developer wants to [pass to the function](https://vaadin.com/docs/latest/components/grid/flow#handling-events), WDYT, is it reasonable to have JsonArray as an input of `invokeLitRendererFunction` function, so developer can pass exact argument values or rely on a default, which passes `Json.createArray()`, meaning no arguments?

Reply:
Good idea. I have never used that, but it makes sense to make it possible for those who do, so I will make that a parameter.